scenes/game_elements/fx/shaker/components/shaker.gd

```diff
@@ -62,6 +66,8 @@ func _ready() -> void:
 func shake(intensity: float = shake_intensity, time: float = duration) -> void:
 	noise.seed = randi()
 	started.emit()
+	if _last_controller_id >= 0:
+		Input.start_joy_vibration(_last_controller_id, 0.5, 0.5, time)
```

Comment:
I've only tested this with the controller I have plugged into my PC which is an 8BitDo Xbox controller clone. The two parameters are the magnitude of the weak and strong motors, each between `0` and `1`. `0.5` is just a number. It feels OK.

https://docs.godotengine.org/en/stable/classes/class_input.html#class-input-method-start-joy-vibration